assets/js/util/i18n.js

```diff
@@ -285,8 +289,43 @@ export const numFmt = ( number, options = {} ) => {
  */
 export const numberFormat = ( number, options = {} ) => {
 	const { locale = getLocale(), ...formatOptions } = options;
+	try {
+		/**
+		 * Per https://github.com/google/site-kit-wp/issues/3255 there have been issues with some versions of Safari
+		 * on some operating systems throwing issues with some parameters in the formatOptions.
+		 *
+		 * If an error is thrown, we remove some troublesome params from the formatOptions object and fallback to no formatting.
+		 *
+		 * This allows us to degrade somewhat gracefully without breaking the dashboard for users of unaffected browsers.
+		 */
+		return new Intl.NumberFormat( locale, formatOptions ).format( number );
+	} catch ( error ) {
+		logOnce( `Unable to: new Intl.NumberFormat( ${ JSON.stringify( locale ) }, ${ JSON.stringify( formatOptions ) } ).format( ${ number } );`, 'warn' );
+
+		// Remove these key/values from formatOptions and try again.
+		const removeFormatOptions = {
```

Comment:
The name is a little confusing here since it's fairly close to `reducedFormatOptions`, it makes the code below a bit difficult to read. How about `unstableFormatOptionValues` ?

Similarly, I think it would be a little easier to read in the loop to have a separate variable for the any-value options (e.g. `unstableFormatOptions`). Then we could just check `unstableFormatOptions.includes( key )` or use an object with the same `*` just to be able to check by key.

Reply:
Addressed in https://github.com/google/site-kit-wp/pull/3620/commits/96d0593c48ce49b0227e787020007e9e966f7a7e